packages/manager/src/features/CloudPulse/Alerts/AlertsListing/AlertListTable.tsx

```diff
@@ -33,14 +33,18 @@ export interface AlertsListTableProps {
    * A boolean indicating whether the alerts are loading
    */
   isLoading: boolean;
+  /**
+   * Callback to scroll till the button element element on page change
```

Comment:
Is the below what was meant here?

```suggestion
   * Callback to scroll to the button element on page change
```